gitlab/config.py

```diff
@@ -20,27 +20,54 @@
 import shlex
 import subprocess
 from os.path import expanduser, expandvars
+from pathlib import Path
```

Comment:
nit: As we discussed in the relative import patch. I think this should be `import pathlib` to improve readability.

Heck the line above should just be `import os` in my opinion.

Reply:
Hmm to me this is such a [standard use of pathlib](https://docs.python.org/3/library/pathlib.html#basic-use) I would keep it least for this PR. We have other instances of it in the repo that would make it inconsistent and many projects do it this way, so I'd maybe keep that for discussion in the other PR :)

Reply:
Yeah lets discuss in the other one. I think that documentation is showing it that way to make the documentation easier to parse.

But from the [Zen of Python](https://www.python.org/dev/peps/pep-0020/)

> Explicit is better than implicit.

As I said this is a "nit" for me. I just want to improve code readability.